packages/frontend/editor-ui/src/components/CanvasChat/future/components/LogsOverviewRow.vue

```diff
@@ -140,43 +144,67 @@ function isLastChild(level: number) {
 			:class="$style.compactErrorIcon"
 		/>
 		<N8nIconButton
+			type="secondary"
+			size="small"
+			icon="play"
+			style="color: var(--color-text-base)"
+			:aria-label="locale.baseText('logs.overview.body.run')"
+			:class="[$style.partialExecutionButton, depth > 0 ? $style.unavailable : '']"
+			@click.stop="emit('triggerPartialExecution', props.data)"
+		/>
+		<N8nIconButton
+			type="secondary"
+			size="small"
+			icon="external-link-alt"
+			style="color: var(--color-text-base)"
+			:class="$style.openNdvButton"
+			:aria-label="locale.baseText('logs.overview.body.open')"
+			@click.stop="emit('openNdv', props.data)"
+		/>
+		<N8nButton
 			v-if="!isCompact || props.data.children.length > 0"
 			type="secondary"
-			size="medium"
-			:icon="props.node.expanded ? 'chevron-down' : 'chevron-up'"
+			size="small"
+			:square="true"
 			:style="{
 				visibility: props.data.children.length === 0 ? 'hidden' : '',
 				color: 'var(--color-text-base)', // give higher specificity than the style from the component itself
 			}"
 			:class="$style.toggleButton"
+			:aria-label="locale.baseText('logs.overview.body.toggleRow')"
 			@click.stop="emit('toggleExpanded', props.node)"
-		/>
+		>
+			<N8nIcon size="medium" :icon="props.node.expanded ? 'chevron-down' : 'chevron-up'" />
+		</N8nButton>
 	</div>
 </template>
 
 <style lang="scss" module>
 .container {
 	display: flex;
-	align-items: stretch;
+	align-items: center;
 	justify-content: stretch;
 	overflow: hidden;
 	position: relative;
 	z-index: 1;
 
+	--row-gap-thickness: 1px;
```

Comment:
Nitpick: Does it make sense to use `--spacing-5xs` token here? It will increase the gap slightly but will be more in line with our design system

Reply:
I'll get feedback from design team soon, let me bring up about this then